### PR TITLE
Update documentation and some class names

### PR DESCRIPTION
**Pull request by artem888**
before:
`USAGE: symbolicator-app <input-file-argument> [--dsym-file <dsym-file>] [--app-name <app-name>] [--arch <arch>] [--json]`

after:
`USAGE: symbolicator <input-file-argument> [--dsym <dsym>] [--app-name <app-name>] [--arch <arch>] [--json]`

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -24,14 +24,14 @@ brew install symbolicator
 
 ```
 # Symbolicate a crash report
-symbolicator YourApp.ips --dsym YourApp.dSYM
+symbolicator --dsym-file YourApp.dSYM YourApp.ips
```
Comment (berikv) [inline README.md:27]: It's not _really_ a file..

File: Sources/Symbolicator/SymbolicatorApp.swift [hunk 2]
```diff
@@ -3,7 +3,7 @@ import ArgumentParser
 import GenericJSON
 
 @main
-struct SymbolicatorApp: ParsableCommand {
+struct Symbolicator: ParsableCommand {
```
Comment (berikv) [inline Sources/Symbolicator/SymbolicatorApp.swift:6]: I'd prefer the `@main` class to be called *App.

File: README.md (hunk 1 above)
Comment (artem888) [reply README.md:27]: true :) lets just call it dsym?

File: Sources/Symbolicator/SymbolicatorApp.swift (hunk 2 above)
Comment (artem888) [reply Sources/Symbolicator/SymbolicatorApp.swift:6]: the reason I renamed was that help documentation was autogenerated like that:

`USAGE: symbolicator-app <input-file-argument> [--dsym-file <dsym-file>] [--app-name <app-name>] [--arch <arch>] [--json]`

**Review by berikv — APPROVED** [review]

**Pull request merged**